### ISSUE TITLE
Show code reviewers column in table list

### ISSUE THREAD

**Issue by ihorpts** [uwolfer/gerrit-intellij-plugin#185]
It would be good to have an option to see if there are more than one code reviewer assigned to the task - to avoid overriding somebody's decision on-the-fly (by merging action for example).

Thanks!

**Comment by uwolfer** [comment]
You can see all assigned reviewers in the change details panel. They are listed as "CC" if they have not done a review yet. Is this what you are looking for?

**Comment by ihorpts** [comment]
@uwolfer yes, true. The only thing is that you have to click through all of them (all commits) to figure out where there are more then one code reviewer. But otherwise you are right...

**Comment by uwolfer** [comment]
@ihorpts: You can find all assigned reviewers quite at the top of the details panel in the row "CC". You do not need to scroll down to the comments. How would you like to display them in the list?

**Comment by ihorpts** [comment]
@uwolfer I was just thinking if it's possible (and make sense) to have additional (configurable) "Code reviewers" column the same as "Owner" column...

**Comment by rupat** [comment]
@uwolfer @ihorpts: in fact this would be a useful feature. In our team we dont set a reviewer so everone can get any review. For now we need to look in the details page of each row to check if already a reviewer is set or we can take the review.

Another option would be to add the possibility to add own filters...
for example we use: -reviewerin:"Registered Users" AND -owner:self AND status:open

Greetings from Cottbus, Germany ;)
